earth_enterprise/src/common/serverdb/map_tile_utils.cpp

```diff
@@ -171,6 +171,7 @@ const MimeType MapTileUtils::kOctetMimeType("application/octet-stream");
 const MimeType MapTileUtils::kJpegMimeType("image/jpeg");
 const MimeType MapTileUtils::kPngMimeType("image/png");
 const MimeType MapTileUtils::kJavascriptMimeType("text/javascript");
+const MimeType MapTileUtils::kJSONMimeType("application/json");
```

Comment:
Style: To be consistent with other variable names, this would be `kJsonMimeType`.